server/app/routes/api/v1/va/triplers.js

```diff
@@ -217,6 +217,11 @@ async function startTriplerConfirmation(req, res) {
     return _400(res, "You entered your phone number as the number of this Vote Tripler. Please try again.");
   }
 
+  let carrierLookup = await carrier(triplerPhone);
```

Comment:
need to define carrier at the top of the file

`import carrier from '../../../../lib/carrier';`

Reply:
![image](https://user-images.githubusercontent.com/1509951/93633577-2b5ef500-f9bd-11ea-9f64-31ed81c3706e.png)

Reply:
Done.